src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -329,6 +372,17 @@ void GcsActorScheduler::HandleRequestWorkerLeaseCanceled(
       << ", cancel type: "
       << rpc::RequestWorkerLeaseReply::SchedulingFailureType_Name(failure_type);
 
+  if (failure_type == rpc::RequestWorkerLeaseReply::SCHEDULING_FAILED) {
```

Comment:
emm, why we need add this logic?

Reply:
This can be removed. Thanks for finding it